drivers/include/net/netdev/ieee802154.h

```diff
@@ -108,6 +108,13 @@ typedef struct {
     uint8_t chan;                           /**< channel */
     uint16_t flags;                         /**< flags as defined above */
     /** @} */
+#ifdef MODULE_GNRC_NETIF2_DEDUP_BCAST
```

Comment:
Module name could be just gnrc_netif2_dedup

Reply:
Rememeber what I said about coffee ^^